README.md

```diff
@@ -16,163 +15,105 @@ You will need to install the following:
+### Locally with environment variables
 
-Example:
+Export these four variables with the values you need.:
 ```
 export POSTGRESQL_URL=postgres://postgres:postgres@localhost:15432/postgres
 export COUCHDB_URL=https://admin:pass@localhost:5984/medic
-export COUCH2PG_SLEEP_MINS=120
 export COUCH2PG_DOC_LIMIT=1000
 export COUCH2PG_RETRY_COUNT=5
 ```
 
-*NOTE:* Currently, cht-couch2pg only runs in node versions 8 , 10 and 12. Later versions of node have been known to fail.+
-
-To run the contents of this repository locally, first, export the above environment variables and then run:  `npm ci && node .`
-
-### Running locally in interactive mode without environment variables
-
-Run it locally in interactive mode: `npm ci && node . -i`
-
-## Running tests through docker-compose
-
-Run tests with:
-
-```bash
-docker-compose  -f docker-compose.test.yml build cht-couch2pg
-docker-compose  -f docker-compose.test.yml run cht-couch2pg grunt test
-```
-
-Run tests in interactive watch mode with: `docker-compose -f docker-compose.test.yml run cht-couch2pg npm run watch`
-
-Run entrypoint script tests with
-
-```bash
-docker-compose -f docker-compose.test.yml run cht-couch2pg ./tests/bash/bats/bin/bats  /app/tests/bash/test.bats
-```
-
-## Running  the docker image
-
-You can run the [docker image](https://hub.docker.com/r/medicmobile/cht-couch2pg) available on docker hub.  You will need to provide the required environment variables.
-
-### Required Environment Variables
-
-#### 1. COUCHDB_URL
-
-This sets the CouchDB instance URL, format: `https://[user]:[password]@[instance-name]:[port]/
-
-#### 2.COUCH2PG_SLEEP_MINS
-
-This sets the number of minutes between synchronization runs.
-
-#### 3. COUCH2PG_DOC_LIMIT
+Then run: `node .`
 
-This sets the document batch size cht-couch2pg fetches from CouchDB every time it querries CouchDB.
+If you want to set and save all possible variables:
 
-#### 4. COUCH2PG_RETRY_COUNT
+1. Copy `sample.env` to `couch2pg.env`
+2. Edit `couch2pg.env` to have all the variables you need.  Note that `POSTGRESQL_URL` shouldn't be edited as it's defined by the variables above it. Be sure to change `POSTGRES_SERVER_NAME` to where ever your postgress server is running.  If it's local, then use `localost`. The default value of `postgres` won't work.
+3. Run: `. ./couch2pg.env&&node .`
 
-This sets the number of times cht-couch2pg will retry synchronizing documents from CouchDB after experiencing an error
+### In docker-compose
 
-#### 5. COUCH2PG_CHANGES_LIMIT
+The simplest way to run with `docker-compose` is to specify the CouchDB instance that your CHT is using.  The compose file will then create a dockerized PostgresSQL instance, connect to the CouchDB server and proceed to download all the data to the PostgresSQL instance:
 
-The number of document ids to fetch per change limit request
+1. Start the docker compose services by running:
+   ```shell
+   export COUCHDB_URL=https://medic:password@192-168-68-26.my.local-ip.co:8442/medic 
```

Comment:
Would be helpful to include a note that points out this address must be accessible from a Docker container (cannot be `localhost`).

Reply:
agreed! fixed in 12b971f